website/client/src/components/groups/groupPlan.vue

```diff
@@ -398,6 +403,10 @@
   .payment-options {
     margin-bottom: 4em;
 
+    h4 {
+      color: #34313a;
```

Comment:
We don't necessarily need to fix all instances in this PR, but if we're adding a new color style we should take the opportunity to import `colors.scss` and use the `$color-XXX` variables.

Reply:
On second thought, there's enough random hex codes in this file that we should probably do it all at one go later... never mind XD